gamedev-iitk/tower-defense
# add more unit tests

**Description:** Added a few more tests to the MovementTests and TowerTests

**Steps to test:**
1.Run using the run all button in test runner
1.The mouse cursor should remain on the game screen at all times during the test.

**Outstanding:**
1. There was one error where comparing the color of indicator to be green it gave two different values with different alpha values(0.3f and 0.212f)
1. Another error i noticed was that if there is and indicator and i instantiate a tower on it then destroy the tower,the indicator color remains red.it does not change to green.i had to retoggle the indicator to change it to green during one of the tests.
